app/views/shared/_github_button.html.erb

```diff
@@ -1,9 +1,9 @@
 <% if @user.password_required? %>
   <% resource_class.omniauth_providers.each do |provider| %>
     <div class="omniauth-login text-center">
-      <%= link_to omniauth_authorize_path(resource_name, provider), class: 'github-login-button page-buttons login btn' do %>
```

Comment:
Now that this partial contains more than one button, can we rename it to `_omniauth_buttons.html.erb` please?